src/app/modules/gb-side-panel-module/accordion-components/gb-cohorts/gb-cohorts.component.ts

```diff
@@ -103,15 +102,7 @@ export class GbCohortsComponent implements AfterViewInit, OnInit {
 
   downloadCohort(e: Event, cohort: Cohort) {
     e.stopPropagation()
-    this.savedCohortsPatientListService.getListStatusNotifier(cohort.name).subscribe(
```

Comment:
why did you un-implement download cohort?

Reply:
I didn't, unfortunately it was not working because not adapted to the geco connector. It was still trying to call non-existing routes of the old medco connector.